en/beatmap_ranking_criteria.md

```diff
@@ -7,18 +7,18 @@ Frequently Asked Questions:
 
 -----------------------
 
-- **Q: Why my requested map is not yet ranked while it's already qualified the criteria?**
+- **Q: Why is my requested map isn't ranked yet while it already meets the qualification for the Ranking  Criteria?**
 - A: Maybe your requested map is not detected by system, you can request again via our Discord. Format: `[REQ ULANG](Link Beatmap)(mode)`.
```

Comment:
Ubah itu 
Please use this format : `[REQ](osu-beatmap-link)(mode)` **Only for Donors**